### PR TITLE
Add firstBookedDateTime to VisitPreviewDto

### PR DESCRIPTION
**Pull request by dhirajshettymoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/VisitDto.kt [hunk 1]
```diff
@@ -59,4 +60,8 @@ class VisitDto(
   @Schema(description = "The visit modified date and time", example = "2018-12-01T13:45:00", required = true)
   @field:NotBlank
   val modifiedTimestamp: LocalDateTime,
+  @Schema(description = "User type", example = "STAFF", required = true)
```
[the remark this answers is not in the document]
Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/VisitDto.kt:63]: yes not needed. I will remove it here. and let #331 add it.

**Review by adaviesMOJ — APPROVED** [review]

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/VisitDto.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/VisitDto.kt]: updated

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/VisitDto.kt [hunk 2]
```diff
@@ -59,4 +59,6 @@ class VisitDto(
   @Schema(description = "The visit modified date and time", example = "2018-12-01T13:45:00", required = true)
   @field:NotBlank
   val modifiedTimestamp: LocalDateTime,
+  @Schema(description = "Date the visit was first booked or migrated", example = "2018-12-01T13:45:00", required = false)
+  val firstBookedDateTime: LocalDateTime? = null,
```
Comment (tpmcgowan) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/VisitDto.kt:63]: Do we need to have this new field on `VisitDto`? (For the requirements on the ticket we only need it on `VisitPreviewDto` as that's what's used on the Visits by date page)

Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/VisitDto.kt:63]: this is the DTO that is returned by visit-scheduler. We then map it into a VisitPreviewDto in the orchestration service

**Pull request merged**